examples/upgrade.py

```diff
@@ -17,7 +22,7 @@ async def main() -> None:
         print("No stable version found")
         return
 
-    async with WLED("10.10.11.54") as led:
+    async with WLED(sys.argv[1]) as led:
```

Comment:
let's remove this is just an example

Reply:
It's easier to run the example to test the behaviour against many different devices in you don't have to edit the file by hand reach each find